_includes/layouts/page.html

```diff
@@ -9,23 +9,30 @@
 <div id="main-content" class="usa-layout-docs usa-section">
   <div class="grid-container">
     <div class="grid-row grid-gap">
-      {% if sidenav == true %}
-      {% include "sidenav.html" %}
-      {% endif %}
+      {%- if sidenav == true -%}
+        {%- include "sidenav.html" -%}
+      {%- endif -%}
       <main class="usa-layout-docs__main desktop:grid-col-9
-                  {% if first_page_style %} first_page {% endif %}
                   usa-prose measure-5 {{tags[0] -}}"
       >
-        {% if title %} 
-          <h1> {{ title }} </h1> 
-        {% endif %}
+        {%- unless page_title_tag -%}
+          {% assign page_title_tag = "h1" %}
+        {%- endunless -%}
+        {%- if title -%}
+          {%- if hidden_guide_title -%}
+          <h1 class="usa-sr-only">{{ hidden_guide_title }}</h1>
+          {%- endif -%}
+          <{{page_title_tag}} class="page-title {{tags[0]}}">{{ title }}</{{page_title_tag}}>
+        {%- endif -%}
         {{ content }}
       </main>
     </div>
   </div>
 </div>
 
-  <script src={{ '/assets/_common/js/current-link.js' | url }}></script>
-{% if script %}
-  <script src="{{ script | url }}"></script>
+{% if tags[0] == 'derisking' %}
+  <script src={{ '/assets/derisking/js/current-link.js' | url }}></script>
```

Comment:
[non-blocking question] Does the Derisking Guide have a unique JS asset, or will other guides need this asset as well?

Reply:
This one is unique to the De-risking guide, the other guides don't use it.